modules/svg/test/index.js

```diff
@@ -0,0 +1 @@
+// TODO add tests
```

Comment:
no need for index, just name your files xyz.test.js and jest will auto detect them.